<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>860</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>860</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cbSort" >
   <property name="geometry" >
    <rect>
     <x>258</x>
     <y>648</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text" >
     <string>Shot Number</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Count</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Ip Current</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Ip Width>0.1k</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Ip Width>100k</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Date</string>
    </property>
   </item>
  </widget>
  <widget class="QSplitter" name="splitter" >
   <property name="geometry" >
    <rect>
     <x>8</x>
     <y>72</y>
     <width>841</width>
     <height>38</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lbTodayShot" >
       <property name="font" >
        <font>
         <family>Luxi Serif</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Today Shot</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTodayIp" >
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>115</red>
             <green>115</green>
             <blue>115</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font" >
        <font>
         <family>Baekmuk Gulim</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="frame" >
        <bool>false</bool>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lbTotalShot" >
       <property name="font" >
        <font>
         <family>Luxi Serif</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Total Shot</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTotalIp" >
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>115</red>
             <green>115</green>
             <blue>115</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font" >
        <font>
         <family>Baekmuk Gulim</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>100 (kA)</string>
       </property>
       <property name="frame" >
        <bool>false</bool>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDateTimeEdit" name="dtStart" >
   <property name="geometry" >
    <rect>
     <x>368</x>
     <y>648</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="displayFormat" >
    <string>yyyy-MM-dd hh:mm:ss</string>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>20</y>
     <width>629</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Luxi Serif</family>
     <pointsize>26</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text" >
    <string>Best Plasma</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSrch" >
   <property name="geometry" >
    <rect>
     <x>658</x>
     <y>646</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Search</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dtEnd" >
   <property name="geometry" >
    <rect>
     <x>508</x>
     <y>648</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="displayFormat" >
    <string>yyyy-MM-dd hh:mm:ss</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckDesc" >
   <property name="geometry" >
    <rect>
     <x>188</x>
     <y>650</y>
     <width>53</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Desc</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leEnd" >
   <property name="geometry" >
    <rect>
     <x>518</x>
     <y>648</y>
     <width>101</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckToday" >
   <property name="geometry" >
    <rect>
     <x>121</x>
     <y>650</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Today</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twBestShot" >
   <property name="geometry" >
    <rect>
     <x>8</x>
     <y>116</y>
     <width>841</width>
     <height>522</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leStart" >
   <property name="geometry" >
    <rect>
     <x>378</x>
     <y>648</y>
     <width>101</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClose" >
   <property name="geometry" >
    <rect>
     <x>762</x>
     <y>646</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
